### ISSUE TITLE
[Fix] The switch to disable animation on play/pause button doesn't work

### ISSUE THREAD

**Issue by JackDandy** [nzbget/nzbget#221]
The switch to disable animation on play/pause button doesn't work.

https://github.com/nzbget/nzbget/blob/develop/webui/index.js#L111 contains a copy/paste error and should read ...

```
this.activityAnimation = this.read('ActivityAnimation', this.activityAnimation) == 'true';
```

this change will fix the highlighted issue.

**Comment by hugbug** [comment]
Thank you!
If you like you can create a PR. Otherwise I'll fix this later tonight.

**Comment by JackDandy** [comment]
Done ... #222 ;)

**Comment by hugbug** [comment]
Fixed via PR #222.
